### PR TITLE
Prepare for distribution on PyPI

### PR DESCRIPTION
**Pull request by rlucioni**
These are a handful of changes necessary to prepare the package for distribution. I've uploaded it along with a source distribution and a wheel to [PyPI](https://pypi.python.org/pypi/edx-auth-backends).

@stephensanchez, could you give this a pass when you're able?

### CODE REVIEW

File: auth_backends/_version.py [hunk 1]
```diff
@@ -0,0 +1 @@
+__version__ = '0.1.1'
```
Comment (stephensanchez) [inline auth_backends/_version.py]: why version 0.1.1? Is that what it was when it was factored out of Insights?

File: setup.py [hunk 2]
```diff
@@ -1,19 +1,30 @@
...
-    long_description=readme(),
+    long_description=(
+        read('README.rst') + '\n\n' +
+        read('HISTORY.rst') + '\n\n' +
+        read('AUTHORS')
+    ),
     classifiers=[
         'Development Status :: 4 - Beta',
         'License :: OSI Approved :: GNU Affero General Public License v3',
+        'Programming Language :: Python',
```
Comment (stephensanchez) [inline setup.py]: Why is Programming Language specified twice?  I'm not familiar with this type of setup.

**Comment by stephensanchez** [comment]
Two small questions but looks good otherwise :+1:

File: auth_backends/_version.py (hunk 1 above)
Comment (rlucioni) [inline auth_backends/_version.py]: I originally published a version 0.1.0 to PyPI with a README which PyPI's reST parser couldn't render properly. I deleted the package in an attempt to fix it, only to realize that if a file is deleted from PyPI, it cannot be re-uploaded again with the same filename. So, in order to re-upload the source distribution and wheel, I had to bump the version. You can read more about the issue in [this thread](https://mail.python.org/pipermail/distutils-sig/2015-January/025687.html) from the distutils mailing list.

File: setup.py (hunk 2 above)
Comment (rlucioni) [inline setup.py]: It's seems to be a convention to list the `'Programming Language :: Python'` classifier, followed by classifiers for each supported Python version. There are no hard and fast rules here, and I wanted to be thorough.

Comment (stephensanchez) [inline setup.py]: Works for me!

**Comment by clintonb** [comment]
:+1:

**Comment by rlucioni** [comment]
I've added a test case covering claim retrieval to bring coverage to 100%. I'm going to go ahead and merge now.

**Pull request merged**